pr_agent/servers/github_app.py

```diff
@@ -48,70 +51,120 @@ async def get_body(request):
     except Exception as e:
         logging.error("Error parsing request body", e)
         raise HTTPException(status_code=400, detail="Error parsing request body") from e
-    body_bytes = await request.body()
-    signature_header = request.headers.get('x-hub-signature-256', None)
-    webhook_secret = getattr(get_settings().github, 'webhook_secret', None)
-    if webhook_secret:
-        verify_signature(body_bytes, webhook_secret, signature_header)
+    if get_settings().github_app.verify_signature:
```

Comment:
isn't it redundant? no webhook_secret = don't verify

Reply:
Good point. I think our issue was with the signiture header missing from the request. If I reorder the flow to only get the header if there is a secret to verify, that should do it and indeed make the configuration toggle I added redundant. Sounds good?